nearcore/src/state_sync.rs

```diff
@@ -79,7 +79,7 @@ impl StateSyncDumper {
         };
 
         // Determine how many threads to start.
-        // TODO: Handle the case of changing the shard layout.
+        // TODO(resharding): Handle the case of changing the shard layout.
```

Comment:
Can we file an issue for this?

Reply:
The convention in resharding is to track outstanding work items with todo's like the above, at least for the time being while the project is frozen.